.github/workflows/gradle.yml

```diff
@@ -10,18 +10,21 @@ on:
     branches: [ main ]
 
 jobs:
+
   build:
 
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up JDK 11
-      uses: actions/setup-java@v2
-      with:
-        java-version: '11'
-        distribution: 'adopt'
-    - name: Grant execute permission for gradlew
-      run: chmod +x gradlew
-    - name: Build with Gradle
-      run: ./gradlew build
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Build with Gradle
+        env:
+          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+        run: | 
+              ./gradlew build
```

Comment:
why build need the secret, the stage should be called "Test"

Reply:
Actually when the pipeline ./gradlew build. Execute: checkStyleTest and test include ApiHttpBinTest and ApiGithubTest

Reply:
Seperate build and test in the last commit